app/controllers/api/v1/products_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module V1
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/ffe479ef97670c7333f37452d55369f226bbc764/app/controllers/api/v1/products_controller.rb#L2"></span>Has the name 'v1'</td>
    </tr>
  </tbody>
</table>